sdk/Cargo.toml

```diff
@@ -134,7 +134,6 @@ tempfile = { version = "3.15", features = ["nightly"] }
 ureq = "2.4.0"
 
 [target.'cfg(any(target_os = "wasi", not(target_arch = "wasm32")))'.dependencies]
-x509-certificate = "0.23.1"
```

Comment:
x509-certificate is no longer referenced by c2pa-rs; that moved entirely to c2pa-crypto.